app/assets/javascripts/components/samples.jsx

```diff
@@ -578,6 +580,42 @@ class Samples extends React.Component {
       </div>
     );
 
+   const filterTissueDropDown = (
+        <div className='dropdown-status-filtering'>
+        <li>
+          <a className="title">
+            <b>Filter tissue</b>
+          </a>
+        </li>
+        <li className="filter-item" data-status="Bronchoalveolar lavage" onClick={ this.handleTissueFilterSelect } ><a data-status="Bronchoalveolar lavage"className="filter-item waiting">Bronchoalveolar lavage</a><i className="filter fa fa-check hidden"></i></li>
+        <li className="filter-item" data-status="Cerebrospinal fluid" onClick={ this.handleTissueFilterSelect }><a data-status="Cerebrospinal fluid" className="filter-item uploading">Cerebrospinal fluid</a><i data-status="Cerebrospinal fluid" className="filter fa fa-check hidden"></i></li>
+        <li className="filter-item" data-status="Nasopharyngeal swab" onClick={ this.handleTissueFilterSelect }><a data-status="Nasopharyngeal swab" className="filter-item complete">Nasopharyngeal swab</a><i  data-status="Nasopharyngeal swab" className="filter fa fa-check hidden"></i></li>
+        <li className="filter-item" onClick={ this.handleTissueFilterSelect } data-status="Plasma" ><a data-status="Plasma" className="filter-item failed">Plasma</a><i data-status="Plasma" className="filter fa fa-check hidden"></i></li>
+        <li className="filter-item" data-status="Serum" onClick={ this.handleTissueFilterSelect }><a data-status="Serum" className="filter-item all">Serum</a><i data-status="Serum"  className="filter all fa fa-check hidden"></i></li>
+        <li className="filter-item" data-status="Solid tissue" onClick={ this.handleTissueFilterSelect }><a data-status="Solid tissue" className="filter-item all">Solid tissue</a><i data-status="Solid tissue" className="filter all fa fa-check hidden"></i></li>
+        <li className="filter-item" data-status="Stool" onClick={ this.handleTissueFilterSelect }><a  data-status="Stool" className="filter-item all">Stool</a><i  data-status="Stool" className="filter all fa fa-check hidden"></i></li>
+        <li className="filter-item" data-status="Synovial fluid" onClick={ this.handleTissueFilterSelect }><a data-status="Synovial fluid" className="filter-item all">Synovial fluid</a><i data-status="Synovial fluid" className="filter all fa fa-check hidden"></i></li>
+        <li className="filter-item" data-status="Whole blood" onClick={ this.handleTissueFilterSelect }><a data-status="Whole blood" className="filter-item all">Whole blood</a><i data-status="Whole blood" className="filter all fa fa-check hidden"></i></li>
+        <li className="divider"/>
+      </div>
+   )
+
```

Comment:
As discussed, I think it would be better to retrieve the different options for the tissue types from PipelineSampleReads.jsx, where it is already hard-coded, rather than hard-coding it here again. For example, you could move the list from line 31 in PipelineSampleReads.jsx to a static method "tissue_types()" that returns the list. Then you would be able to access it here as PipelineSampleReads.tissue_types(). (Or you could move the list into a helper.) Then, here, you would just iterate over the list rather than writing everything out.